src/sql/workbench/contrib/query/browser/gridPanel.ts

```diff
@@ -269,12 +269,13 @@ export class GridPanel extends Disposable {
 
 		for (let i = this.tables.length - 1; i >= 0; i--) {
 			if (this.tables[i].id === tableid) {
-				this.tables[i].state.maximized = true;
-				this.maximizedGrid = this.tables[i];
-				continue;
+				const selectedTable = this.tables[i];
+				selectedTable.state.maximized = true;
+				this.maximizedGrid = selectedTable;
+				this.scrollableView.clear();
+				this.scrollableView.addViews([selectedTable]);
+				break;
```

Comment:
Just to double confirm the change from continue to break is on purpose?

Reply:
yeah, just making it more efficient

Reply:
@abist FYI - this could be simplified even more by just using the result from the find call above instead of searching a second time.